examples/medplum-patient-intake-demo/src/bots/core/intake-utils.ts

```diff
@@ -136,3 +210,123 @@ export function addLanguage(patient: Patient, valueCoding: Coding, preferred: bo
 
   patient.communication = patientCommunications;
 }
+
+export async function addCoverage(
+  medplum: MedplumClient,
+  patient: Patient,
+  answers: Record<string, QuestionnaireResponseItemAnswer>
+): Promise<void> {
+  const payor = await medplum.createResource({
+    resourceType: 'Organization',
+    name: answers['insurance-provider']?.valueString,
```

Comment:
We should update the questionnaire and response to use `valueReference`. I will update your project with our PayerDirectory, which has a link to all kinds of existing `Oganization` resources fo rinsurance companies. You should not have to create this one